rhizome/host/bin/prep_host.rb

```diff
@@ -22,11 +22,28 @@
 safe_write_to_file("/etc/hosts", File.read("/etc/hosts").gsub(original_hostname, hostname))
 r "sudo hostnamectl set-hostname " + hostname
 
-if env_type == "production"
-  bashrc_content = File.read("/root/.bashrc")
-  colored_prompt_code = '\e[0;41m[\u@\h \W]\$ \e[m'
-  safe_write_to_file("/root/.bashrc", "#{bashrc_content}\n PS1='#{colored_prompt_code}'")
-end
+color_code = (env_type == "production") ? "\e[31m" : "\e[32m"
+
+reset_color_code = "\e[0m"
+
+bashrc_content = File.read("/root/.bashrc")
+colored_prompt_code = color_code + '[\u@\h \W]\$ ' + reset_color_code
+safe_write_to_file("/root/.bashrc", "#{bashrc_content}\n PS1='#{colored_prompt_code}'")
+
+motd_message = <<~'MOTD_MESSAGE'
+ _   _ ____ ___ ____ _     ___  _   _ ____
```

Comment:
For the record, which font did you use in figlet? I looked various figlet fonts using [this tool](https://www.askapache.com/online-tools/figlet-ascii/). It appears to be `standard`. There are, however, more readable fonts. If I don't know it's UBICLOUD, I can't directly identify it as such.

### lowercase:

```
+--- big

       _     _      _                 _ 
      | |   (_)    | |               | |
 _   _| |__  _  ___| | ___  _   _  __| |
| | | | '_ \| |/ __| |/ _ \| | | |/ _` |
| |_| | |_) | | (__| | (_) | |_| | (_| |
 \__,_|_.__/|_|\___|_|\___/ \__,_|\__,_|
                                        
                                        
+--- doom

       _     _      _                 _ 
      | |   (_)    | |               | |
 _   _| |__  _  ___| | ___  _   _  __| |
| | | | '_ \| |/ __| |/ _ \| | | |/ _` |
| |_| | |_) | | (__| | (_) | |_| | (_| |
 \__,_|_.__/|_|\___|_|\___/ \__,_|\__,_|
                                        
                                        

+--- larry3d

        __                  ___                       __     
       /\ \      __        /\_ \                     /\ \    
 __  __\ \ \____/\_\    ___\//\ \     ___   __  __   \_\ \   
/\ \/\ \\ \ '__`\/\ \  /'___\\ \ \   / __`\/\ \/\ \  /'_` \  
\ \ \_\ \\ \ \L\ \ \ \/\ \__/ \_\ \_/\ \L\ \ \ \_\ \/\ \L\ \ 
 \ \____/ \ \_,__/\ \_\ \____\/\____\ \____/\ \____/\ \___,_\
  \/___/   \/___/  \/_/\/____/\/____/\/___/  \/___/  \/__,_ /
                                                             
                                                             
                                                                        
                                                                       

+--- letters

        bb      iii        lll                     dd 
uu   uu bb            cccc lll  oooo  uu   uu      dd 
uu   uu bbbbbb  iii cc     lll oo  oo uu   uu  dddddd 
uu   uu bb   bb iii cc     lll oo  oo uu   uu dd   dd 
 uuuu u bbbbbb  iii  ccccc lll  oooo   uuuu u  dddddd 
                                                      



+--- ogre

       _     _      _                 _ 
 _   _| |__ (_) ___| | ___  _   _  __| |
| | | | '_ \| |/ __| |/ _ \| | | |/ _` |
| |_| | |_) | | (__| | (_) | |_| | (_| |
 \__,_|_.__/|_|\___|_|\___/ \__,_|\__,_|
                                        


+--- slant

         __    _      __                __
  __  __/ /_  (_)____/ /___  __  ______/ /
 / / / / __ \/ / ___/ / __ \/ / / / __  / 
/ /_/ / /_/ / / /__/ / /_/ / /_/ / /_/ /  
\__,_/_.___/_/\___/_/\____/\__,_/\__,_/   
                                          


+--- standard

       _     _      _                 _ 
 _   _| |__ (_) ___| | ___  _   _  __| |
| | | | '_ \| |/ __| |/ _ \| | | |/ _` |
| |_| | |_) | | (__| | (_) | |_| | (_| |
 \__,_|_.__/|_|\___|_|\___/ \__,_|\__,_|
                                        


```

### uppercase:

```
+--- big

 _    _ ____ _____ _____ _      ____  _    _ _____  
| |  | |  _ \_   _/ ____| |    / __ \| |  | |  __ \ 
| |  | | |_) || || |    | |   | |  | | |  | | |  | |
| |  | |  _ < | || |    | |   | |  | | |  | | |  | |
| |__| | |_) || || |____| |___| |__| | |__| | |__| |
 \____/|____/_____\_____|______\____/ \____/|_____/ 
                                                    
                                                    

+--- doom

 _   _______ _____ _____  _     _____ _   _______ 
| | | | ___ \_   _/  __ \| |   |  _  | | | |  _  \
| | | | |_/ / | | | /  \/| |   | | | | | | | | | |
| | | | ___ \ | | | |    | |   | | | | | | | | | |
| |_| | |_/ /_| |_| \__/\| |___\ \_/ / |_| | |/ / 
 \___/\____/ \___/ \____/\_____/\___/ \___/|___/  
                                                  
                                                  

+--- larry3d

 __  __  ____    ______   ____     __       _____   __  __  ____      
/\ \/\ \/\  _`\ /\__  _\ /\  _`\  /\ \     /\  __`\/\ \/\ \/\  _`\    
\ \ \ \ \ \ \L\ \/_/\ \/ \ \ \/\_\\ \ \    \ \ \/\ \ \ \ \ \ \ \/\ \  
 \ \ \ \ \ \  _ <' \ \ \  \ \ \/_/_\ \ \  __\ \ \ \ \ \ \ \ \ \ \ \ \ 
  \ \ \_\ \ \ \L\ \ \_\ \__\ \ \L\ \\ \ \L\ \\ \ \_\ \ \ \_\ \ \ \_\ \
   \ \_____\ \____/ /\_____\\ \____/ \ \____/ \ \_____\ \_____\ \____/
    \/_____/\/___/  \/_____/ \/___/   \/___/   \/_____/\/_____/\/___/ 
                                                                      
                                                                      

+--- slant

   __  ______  ____________    ____  __  ______ 
  / / / / __ )/  _/ ____/ /   / __ \/ / / / __ \
 / / / / __  |/ // /   / /   / / / / / / / / / /
/ /_/ / /_/ // // /___/ /___/ /_/ / /_/ / /_/ / 
\____/_____/___/\____/_____/\____/\____/_____/  
                                                


+--- small

 _   _ ___ ___ ___ _    ___  _   _ ___  
| | | | _ )_ _/ __| |  / _ \| | | |   \ 
| |_| | _ \| | (__| |_| (_) | |_| | |) |
 \___/|___/___\___|____\___/ \___/|___/ 
                                        
                                           


+--- standard

 _   _ ____ ___ ____ _     ___  _   _ ____  
| | | | __ )_ _/ ___| |   / _ \| | | |  _ \ 
| | | |  _ \| | |   | |  | | | | | | | | | |
| |_| | |_) | | |___| |__| |_| | |_| | |_| |
 \___/|____/___\____|_____\___/ \___/|____/ 
```

Reply:
I've tried some but couldn't determine which one looks better, then decided to use standard one. If you have any preference feel free to suggest it.